cmd/promtool/testdata/prettify-rules.output.yml

```diff
@@ -0,0 +1,10 @@
+groups:
+    - name: Test
```

Comment:
This output is from encoding `yaml.Node`. The indentation looks strange. I am not sure how to fix this.

Reply:
@Harkishen-Singh `SetIndent(2)` on the v3 encoder will change the indent to 2 spaces rather than 4. However, to prevent the sequence indicator being included in the indentation you'd need to pull in https://github.com/go-yaml/yaml/pull/753, probably easiest way would be to switch to the upcoming fork being used in kube/kustomize  https://github.com/kubernetes-sigs/yaml/issues/72

Reply:
@Harkishen-Singh `SetIndent(2)` on the v3 encoder will change the indent to 2 spaces rather than 4. However, to prevent the sequence indicator being included in the indentation you'd need to pull in https://github.com/go-yaml/yaml/pull/753, probably easiest way would be to switch to the upcoming fork being used in kube/kustomize  https://github.com/kubernetes-sigs/yaml/issues/72